SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSubviews.cpp

```diff
@@ -0,0 +1,219 @@
+#include "Interop/ActorSubviews.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/InitialOnlyFilter.h"
+#include "Schema/Restricted.h"
+#include "Schema/Tombstone.h"
+#include "Schema/UnrealMetadata.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewCoordinator.h"
+
+namespace SpatialGDK
+{
+namespace ActorSubviews
+{
+namespace MainActorSubviewSetup
```

Comment:
```suggestion
{

namespace MainActorSubviewSetup
```

Reply:
I'm afraid this makes clang-format unhappy :(